raft/test/raft_test.cpp

```diff
@@ -189,6 +191,87 @@ namespace
         peer["uuid"] = TEST_NODE_UUID;
         return peer;
     }
+
+
+    database_header*
```

Comment:
I'm surprised protobuf doesn't use smart pointers.

Reply:
Me too, that threw me for a loop. I didn't think to look at the doc for the "set_allocated_" type functions, so when I used &variable on variables allocated on the stack, when a bzn_msg went out of scope, there was an exception. To make that work you need to match each "set_allocated_" with a "release_", otherwise, do pointers and allow bzn_msg to clean up for you by calling the release...